src/shared/components/lazyMobXTable/utils.ts

```diff
@@ -11,13 +11,39 @@ export function maxPage(displayDataLength: number, itemsPerPage: number) {
 
 export function isQueriedStudyAuthorized(study: CancerStudy) {
     return (
-        !getServerConfig().skin_show_unauthorized_studies ||
-        (getServerConfig().skin_show_unauthorized_studies &&
-            //@ts-ignore
-            study.isAuthorized !== false)
+        !getServerConfig().skin_home_page_show_unauthorized_studies ||
+        (getServerConfig().skin_home_page_show_unauthorized_studies &&
+            study.readPermission !== false)
     );
 }
 
+export function hasJsonPathPlaceholders(message: String) {
```

Comment:
@dianab0 this seems like an odd place for these functions.  Like, what does this have to do with lazyMoxTable?

Reply:
same could be said for isQueriedSutdyAUthorized above (i realize you didn't put it here but maybe we can move it somewhere more sensible

Reply:
@alisman I moved the json path methods to a new Utils file src/shared/lib/JsonPathUtils.ts and the isQueriedStudyAuthorized to the src/pages/studyView/StudyViewUtils.tsx as it is used in the StudyViewPageStore.tsx only.